src/contexts/DanteFinance/DanteFinanceProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { useWallet } from 'use-wallet';
+
+import config from '../../config';
+import DanteFinance from '../../dante-finance';
+
+export interface DanteFinanceContext {
+  danteFinance?: DanteFinance;
+}
+
+export const Context = createContext<DanteFinanceContext>({
+  danteFinance: null!,
+});
+
+export const DanteFinanceProvider: React.FC = ({ children }) => {
```

Comment:
Do not use `React.FC`. https://github.com/facebook/create-react-app/pull/8177

Create components with the following syntax:
```
  // ....
}

export function TestCmp(props: TestCmpProps): JSX.Element {
  // ...
}
```